docs/running-on-yarn.md

```diff
@@ -495,6 +468,20 @@ To use a custom metrics.properties for the application master and executors, upd
   name matches both the include and the exclude pattern, this file will be excluded eventually.
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.report.interval</code></td>
+  <td>1s</td>
+  <td>
+  Interval between reports of the current app status in Yarn cluster mode.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.yarn.services</code></td>
+  <td>Nil</td>
```

Comment:
As far the user is concerned, I think the best way to express the default is "(none)". `Nil` is an implementation detail